streams/src/main/java/org/apache/kafka/streams/processor/internals/CompositeRestoreListener.java

```diff
@@ -32,7 +33,7 @@
     public static final NoOpStateRestoreListener NO_OP_STATE_RESTORE_LISTENER = new NoOpStateRestoreListener();
     private final BatchingStateRestoreCallback internalBatchingRestoreCallback;
     private final StateRestoreListener storeRestoreListener;
-    private StateRestoreListener globalRestoreListener = NO_OP_STATE_RESTORE_LISTENER;
+    private StateRestoreListener userRestoreListener = NO_OP_STATE_RESTORE_LISTENER;
```

Comment:
I'm in agreement with the renaming to `userRestoreListener` as this is an internal class and the name "global" may be confusing. 

This PR left the name as is in `KafkaStreams` so users can know the restore
listener is global for the application, which still makes sense to me.

Reply:
Exactly. That was the intention.